index.html

```diff
@@ -22,5 +22,29 @@
         </div>
       </nav>
     </header>
+    <main class="main">
+      <h1 class="h1">I'M Charle Peter. Glad to see you!</h1>
+      <p class="p">
+        I’m a software developer! I can help you build a product , feature or
+        website Look through some of my work and experience! If you like what
+        you see and have a project you need coded, don’t hestiate to contact me.
+      </p>
+      <h4 class="h4-connect">LET'S CONNECT</h4>
+      <div class="flex item-center">
+        <a class="icon-tag" href="#"
+          ><img src="/image/twitter.png" alt="twitter"
+        /></a>
+        <a class="icon-tag" href="#"
+          ><img src="/image/linkedin.png" alt="linkedin"
+        /></a>
+        <a class="icon-tag" href="#"><img src="/image/m.png" alt="m" /></a>
+        <a class="icon-tag" href="#"
+          ><img src="/image/git.png" alt="Github"
+        /></a>
+        <a class="icon-tag" href="#"
+          ><img src="/image/angelist.png" alt="angelist"
+        /></a>
+      </div>
```

Comment:
- [ ] Please note that the headline section should be wrapped in the `<section>` tag , please fix that 

> the section tag represents a generic standalone section of the document, so try always to use it. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) for more info.